src/backend/routers/students.ts

```diff
@@ -17,9 +16,15 @@ export const studentProcedures = {
 
       return result;
     }),
+  //* this function gets students for the current logged in case manager
```

Comment:
🐧 the convention in JS/TS is to comment functions like this (called JSDoc):
![Screenshot 2023-05-25 at 9 09 10 PM](https://github.com/sfbrigade/compass/assets/7410405/b7ed28c2-6563-4252-b0a8-3a6102bc0e3b)

if that format is followed, then the comment is shown when hovering over the function in frontend routes:

![Screenshot 2023-05-25 at 9 09 21 PM](https://github.com/sfbrigade/compass/assets/7410405/eff04c4e-38b5-41ca-847d-25b8d3c0a900)

Reply:
Noted. This might be useful to be added on the Wiki @tessathornberry.